benchmarks/README.rst

```diff
@@ -0,0 +1,40 @@
+..  -*- rst -*-
+
+===================
+NiFreeze benchmarks
+===================
+
+Benchmarking NiFreeze with Airspeed Velocity.
+
+Usage
+-----
+
+Airspeed Velocity manages building and Python environments by itself,
+unless told otherwise. To run the benchmarks, you do not need to
+install a development version of NiFreeze to your current Python environment.
```

Comment:
Semantic line breaks:
```suggestion
unless told otherwise.
To run the benchmarks, you do not need to install
a development version of *NiFreeze* on your current
*Python* environment.
```

Reply:
We should add an rst linter: different people will disagree on this.

Reply:
> A linter cannot deal with semantics...

OK, valid point, but my point is to reduce reviewing and maintenance burden, and avoiding inconsistencies across all NiPreps. It is even hard to enforce this within a single repository. I'd rather adopt a linter and content ourselves with however that linter breaks lines, even if they are not semantic breaks. Requires effort to get used to probably, but involves less maintenance.

https://pypi.org/project/restructuredtext-lint/
https://pypi.org/project/sphinx-lint/

Maybe there are better ones as well.

> The question is whether you or other maintainer disagrees :)

:white_flag: if I am the only one disagreeing? Topic for tech call?